### ISSUE TITLE
syncing with the network

### ISSUE THREAD

**Issue by MAECTPO40** [langerhans/dogecoin-wallet-new#60]
2 days no synchronization with the network. what is the problem?

**Comment by zzudwa** [comment]
It's been a year since your question and here we get same problem again. Not syncing already for 6 weeks

**Comment by SikoSoft** [comment]
Yup... a year later and this app has only gotten more unreliable.

Google needs to remove it from the Play store because many people are being robbed by its issues.

**Comment by makarchuk2020** [comment]
When payment. How connect to network? Stock in payment "not send to network"?

**Comment by btc2dogecom** [comment]
> When payment. How connect to network? Stock in payment "not send to network"?

need wait for the syncing fully to updated. then the payment can broadcast to the network.

**Comment by shingoni** [comment]
It synced with me but the payment fails to transmit, been like that for a week.

**Comment by shingoni** [comment]
@langerhans please help us solve this
